### PR TITLE
[14.0][ADD]purchase_work_acceptance_evaluation

### PR DESCRIPTION
**Pull request by PrapassornS**

### CODE REVIEW

**Comment by PrapassornS** [comment]
@kittiu please review

File: purchase_work_acceptance_evaluation/__manifest__.py
Comment (kittiu) [inline purchase_work_acceptance_evaluation/__manifest__.py:20]: Please change this to `kittiu` for me. Thanks.

Comment (kittiu) [inline purchase_work_acceptance_evaluation/__manifest__.py]: Please help change the maintainer.
```suggestion
    "maintainers": ["kittiu"],
```

File: purchase_work_acceptance_evaluation/models/work_acceptance.py
Comment (kittiu) [inline purchase_work_acceptance_evaluation/models/work_acceptance.py]:
```suggestion
```

**Comment by kittiu** [comment]
Please rebase.
Your branch still don't have `purchase_work_acceptance` yet. **I am surprised why this pass test**

**Comment by PrapassornS** [comment]
@kittiu please review

**Comment by kittiu** [comment]
Please rebase first.

**Comment by PrapassornS** [comment]
Rebased, Please review. @kittiu

File: purchase_work_acceptance_evaluation/tests/test_purchase_work_acceptance_evaluation.py
Comment (kittiu) [inline purchase_work_acceptance_evaluation/tests/test_purchase_work_acceptance_evaluation.py:29]: Should be link this
```suggestion
        with Form(self.Config) as c:
            c.group_enable_eval_on_wa = True
        c.save()
```

**Review by kittiu — COMMENTED** [review]
It is not work now. We should not use @api.contraints to check. Other wise, if we set state_required = 'draft', we will never be able to save the drafted WA.

I refactor all, please help review.

https://github.com/OCA/purchase-workflow/pull/1021/commits/54660349cf59f9d5ad51782b9ab566b0d1f71536

@PrapassornS

**Comment by kittiu** [comment]
Ready for review. @newtratip

**Comment by newtratip** [comment]
@PrapassornS, Please fix test script to coverage code.

**Comment by PrapassornS** [comment]
@newtratip Please review.

**Comment by newtratip** [comment]
active field not neccessary to show on `Work Acceptance Evaluation` (Fixed https://github.com/OCA/purchase-workflow/pull/1021/commits/d0ab256b818ba918dd820caa7000bce683edd56b)

We can Archive by click `Action -> Archive`, On the document will show red ribbon `ARCHIVED` on the top right corner

We can Unarchive by click `Action -> Unarchive`

**Review by newtratip — APPROVED** [review]
Functional Review/Technical Review :+1:

**Comment by newtratip** [comment]
@kittiu, Please approve/merge if all fixed are okay.

**Review by kittiu — APPROVED** [review]

**Pull request merged**
